updates_checklist.yml

```diff
@@ -10,3 +10,10 @@ v3.2:
         '#description_successful': '<p>The replace button has been successfully changed.</p>'
         '#description_failed': '<p><strong>The replace button could not be changed to the remove button. Please check the image paragraphs widget settings.</strong><p>'
         configure_page: { '#text': 'Paragraph configuration', '#url': 'internal:/admin/structure/paragraphs_type/image/form-display' }
+    thunder_update_8303:
+        '#title': 'Enable "Autosave Form" module'
+        '#description': '<p>The "Autosave Form" module provides automatic perodical saving on content entity forms.</p>'
+        '#description_successful': '<p>The "Autosave Form" module has been installed.</p>'
+        '#description_failed': '<p><strong>Installation failed</strong>, visit the extension page and try to enable the module manually.<p>'
+        configure_page: { '#text': 'Autosave Form configuration', '#url': 'internal:/admin/config/content/autosave_form' }
+        module_page: { '#text': 'Autosave Form module page', '#url': 'https://www.drupal.org/project/autosave_form' }
```

Comment:
Sorry, for not explained properly explained comment last time. It's nice that you have added both links to config and module page. What I suggested is to add `/admin/modules` - because we are explaining to user `..., visit the extension page and try to enable the module manually.`, that's why we should provide a link to that page.